native_client/java/libdeepspeech/src/androidTest/java/org/mozilla/deepspeech/libdeepspeech/test/BasicTest.java

```diff
@@ -149,4 +151,19 @@ public void loadDeepSpeech_sttWithMetadata_withLM() {
         assertEquals("she had your dark suit in greasy wash water all year", decoded);
         m.freeModel();
     }
+
+    @Test
+    public void loadDeepSpeech_HotWord() {
+        DeepSpeechModel m = new DeepSpeechModel(modelFile);
+
+        for(int i=0; i< word.size(); i++){
```

Comment:
nit: space around `=`, `<` and after `)`

Reply:
`for( int i=0; i < word.size(); i++ )` do you mean like this?

Reply:
no, read what I said

Reply:
`for( int i = 0; i < word.size(); i++ ) ` I think this is fine.